diff --git a/node_modules/abstract-sdk/abstract-sdk.d.ts b/node_modules/abstract-sdk/abstract-sdk.d.ts
index a940c1b..2618192 100644
--- a/node_modules/abstract-sdk/abstract-sdk.d.ts
+++ b/node_modules/abstract-sdk/abstract-sdk.d.ts
@@ -1838,10 +1838,10 @@ type WebhookDelivery = {
   webhookId: string
 };
 
-type ErrorData = {|
+type ErrorData = {
   path: string,
   body: mixed
-|};
+};
 
 type ErrorMap = {
   [mode: string]: Error
