diff --git a/node_modules/abstract-sdk/abstract-sdk.d.ts b/node_modules/abstract-sdk/abstract-sdk.d.ts
index 86d89a0..3c8aae9 100644
--- a/node_modules/abstract-sdk/abstract-sdk.d.ts
+++ b/node_modules/abstract-sdk/abstract-sdk.d.ts
@@ -1899,10 +1899,10 @@ type WebhookDelivery = {
   webhookId: string
 };
 
-type ErrorData = {|
+type ErrorData = {
   path: string,
   body: mixed
-|};
+};
 
 type ErrorMap = {
   [mode: string]: Error
